### PR TITLE
Added a command TurnOnClimbMode

### PR DESCRIPTION
**Pull request by JalenEvans**
TurnOnClimbMode switches a boolean flag that allows the climb to worl.
Without it on, climb does not run. I also commented the intakeBtn and
uncommented deployBtn because intakeBtn was only menat for testing.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ClimbSubsystem.java [hunk 1]
```diff
@@ -129,6 +131,13 @@ public void zeroRotatingArm()
     secondStageMotor2.setSelectedSensorPosition(0, 0, 0);
   }
 
+  /**
+   * Changes the current status of climbMode to th eopposite of what it was
+   */
+  public void setClimbMode() {
+    climbMode = !climbMode;
+  }
```
Comment (hazeltorek) [inline src/main/java/frc/robot/subsystems/ClimbSubsystem.java]: this is a toggle and not a set

**Review by hazeltorek — CHANGES_REQUESTED** [review]

**Review by 20MatthewNeal — APPROVED** [review]

File: src/main/java/frc/robot/subsystems/ClimbSubsystem.java (hunk 1 above)
Comment (j-hunter) [reply src/main/java/frc/robot/subsystems/ClimbSubsystem.java]: Also, be sure to still have a set and get method in addition to a toggle

File: src/main/java/frc/robot/commands/climb/TurnOnClimbMode.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  public void execute() {
+    climb.setClimbMode();
+    intake.deployIntake();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    climb.setClimbMode();
+    intake.retractIntake();
```
Comment (j-hunter) [inline src/main/java/frc/robot/commands/climb/TurnOnClimbMode.java]: This needs to run in an execute loop to retract the intake

**Review by j-hunter — CHANGES_REQUESTED** [review]

**Review by j-hunter — APPROVED** [review]

**Review by j-hunter — APPROVED** [review]

**Review by hazeltorek — APPROVED** [review]
fine i guess

File: src/main/java/frc/robot/subsystems/ClimbSubsystem.java [hunk 3]
```diff
@@ -129,6 +131,28 @@ public void zeroRotatingArm()
...
+  /**
+   * Changes the current status of climbMode to the desired state
+   */
+  public void setClimbMode(boolean mode) {
+    climbMode = mode;
+  }
+
+  /**
+   * Returns climbMode
+   * @return boolean
```
Comment (hazeltorek) [inline src/main/java/frc/robot/subsystems/ClimbSubsystem.java]: "true if climb is active, false otherwise"

**Review by hazeltorek — CHANGES_REQUESTED** [review]

**Review by hazeltorek — APPROVED** [review]

**Comment by hazeltorek** [comment]
merged as part of #45

**Pull request merged**